code/__DEFINES/atmospherics.dm

```diff
@@ -1,3 +1,9 @@
+#define R_IDEAL_GAS_EQUATION	8.31	//kPa*L/(K*mol)

```

Comment:
this is just a number

Reply:
this is a constant not an equation lmao

Reply:
the define is poorly named, I'll rename it to `R_IDEAL_GAS_CONSTANT` after this pr